ScoutSuite/providers/aws/resources/iam/users.py

```diff
@@ -16,5 +16,6 @@ def _parse_user(self, raw_user):
         raw_user['id'] = raw_user.pop('UserId')
         raw_user['name'] = raw_user.pop('UserName')
         raw_user['arn'] = raw_user.pop('Arn')
-
+        if (len(raw_user['tags']['Tags']) > 0):
```

Comment:
If you don't need _get_and_set_user_tags, tags will be accessible directly in raw_user['Tags']

Reply:
I was triying to access ['Tags'] from HTML, but without this conversion was imposible for me